app/assets/fonts/icomoon.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
+<svg xmlns="http://www.w3.org/2000/svg">
+<metadata>
+<json>
+<![CDATA[
+{
+	"fontFamily": "icomoon",
+	"majorVersion": 1,
+	"minorVersion": 0,
+	"version": "Version 1.0",
+	"fontId": "icomoon",
+	"psName": "icomoon",
+	"subFamily": "Regular",
+	"fullName": "icomoon",
+	"description": "Font generated by IcoMoon."
+}
+]]>
+</json>
+</metadata>
+<defs>
+<font id="icomoon" horiz-adv-x="1024">
+<font-face units-per-em="1024" ascent="960" descent="-64" />
+<missing-glyph horiz-adv-x="1024" />
+<glyph unicode="&#x20;" horiz-adv-x="512" d="" />
+<glyph unicode="&#xe900;" glyph-name="chevron-thin-down" data-tags="chevron-thin-down" d="M891.802 660.019c13.926 13.722 36.301 13.722 50.125 0s13.875-35.891 0-49.613l-404.89-400.896c-13.824-13.722-36.198-13.722-50.125 0l-404.89 400.896c-13.824 13.722-13.824 35.891 0 49.613 13.875 13.722 36.301 13.722 50.125 0l379.853-365.619 379.802 365.619z" />
+<glyph unicode="&#xe901;" glyph-name="close" data-tags="x" horiz-adv-x="640" d="M640 640l-128 128-192-192-192 192-128-128 192-192-192-192 128-128 192 192 192-192 128 128-192 192 192 192z" />
+<glyph unicode="&#xea10;" glyph-name="checkmark" data-tags="checkmark, tick, correct, accept, ok" d="M864 832l-480-480-224 224-160-160 384-384 640 640z" />
+</font></defs></svg>
```

Comment:
Same thing here about putting this in vendor. I'm pretty sure this will work if it's in vendor/assets too.